### ISSUE TITLE
Progress callback doesn't stop on abort

### ISSUE THREAD

**Issue by nivm** [TTLabs/EvaporateJS#188]
The progress callback doesn't stop on abort, or on already cached files.
Resulting in endless calling for the progress with 1 value.

line 
```javascript
function setStatus(s) {
                if (s === COMPLETE || s === ERROR || s === CANCELED) {
                    clearInterval(progressTotalInterval);
                    clearInterval(progressPartsInterval);
                }
                me.status = s;
                me.onStatusChange();
           }
```

setStatus function doesn't clearInterval on status abort.

Will create a matching pull request.

**Comment by bikeath1337** [comment]
Thanks @nivm , I believe these are fixed in #184, which is ready for a release candidate (pause and resume).

**Comment by nivm** [comment]
OK, closing the issue. 

P.S.

@bikeath1337  - you are doing a great job on this project.
